PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/networking/BaseNetworkTask.java

```diff
@@ -52,6 +62,7 @@
     protected static final String CONTENT_TYPE_HEADER = "Content-Type";
     protected static final String CONTENT_TYPE_HEADER_VALUE = "application/json";
 
+    public boolean isStatusRequest = false;
```

Comment:
We shouldn't add specific login into the _general_ class. `BaseNetworkTask` shouldn't know the request details.